assets/js/components/settings/settings-module.js

```diff
@@ -91,26 +92,19 @@ class SettingsModule extends Component {
 
 	async activateOrDeactivate() {
 		try {
-			const { active } = this.state;
-			const newActiveState = ! active;
-
 			this.setState( { isSaving: true } );
 
 			await activateOrDeactivateModule(
 				data,
 				this.props.slug,
-				newActiveState
+				false
```

Comment:
Why set this to always be `false`? It seems this function will always deactivate rather than handling both as it did before.